front/mailing/20240308-weekly-update.ts

```diff
@@ -113,7 +113,7 @@ JOIN "memberships" "m" ON "u"."id" = "m"."userId"
 JOIN "workspaces" "w" ON "m"."workspaceId" = "w"."id"
 JOIN "subscriptions" "s" ON "w"."id" = "s"."workspaceId"
 WHERE "s"."status" = 'active'
-AND "m"."role" != 'revoked';
+AND ("m"."startAt" <= NOW()) AND ("m"."endAt" IS NULL OR "m"."endAt" >= NOW());
```

Comment:
We still use that migration or no ?

Reply:
There's definitely still a chance we'll use it I think.